rdt/transformers/id.py

```diff
@@ -10,6 +10,7 @@
 from rdt.transformers.utils import strings_from_regex
 
 LOGGER = logging.getLogger(__name__)
+_CARDINALITY_RULE_SENTINEL = object()
```

Comment:
I spoke with @npatki and I think we can skip this part. Instead, we can always default to whatever the cardinality rule is set to. That is similar to what we do here
https://github.com/sdv-dev/RDT/blob/5d396e01ee67d6cd3e0ad1f0638dd75983f6ca4e/rdt/transformers/pii/anonymizer.py#L124-L131